### ISSUE TITLE
Description is a required field for responses

### ISSUE THREAD

**Issue by dbanty** [poem-web/poem#86]
## Expected Behavior

All responses, regardless of how they're derived, will include a description. Specifically, [this bit of the spec](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#response-object) specifies it as a required field.

## Actual Behavior

No description is included by default, and there were no (obvious) examples of how to add one. I did find out from poking around that simply adding a comment above a response variant will add a description, but this should be called out in the book (or API docs somewhere, though not sure where that'd fit).

## Steps to Reproduce the Problem

Use a response like this:

```rust
#[derive(Debug, ApiResponse)]
enum Response {
    #[oai(status = 200)]
    Ok(
        Json<u64>,
    ),
}
```

Notice that the OpenAPI document has the response documented like this:

```json
"responses": {
  "200": {
    "content": {
      "application/json": {
        "schema": {
          "type": "integer",
          "format": "uint64"
        }
      }
    }
  }
}
```

Some tooling (including the generator I maintain 😬) will fail schema validation of the document when parsing this.

## Specifications

```toml
poem = { version = "1.0.24", features = ["multipart", "tempfile"] }
poem-openapi = "1.0.20"
```

- Subsystem: openapi

**Comment by dbanty** [comment]
Happy to take a crack at solving this, maybe with just a default empty string instead of omitted?

**Comment by sunli829** [comment]
> Happy to take a crack at solving this, maybe with just a default empty string instead of omitted?

Yes, I also like this,  welcome to PR. 🙂

**Comment by sunli829** [comment]
Fixed in `v1.0.27` 😁
